scipy/optimize/_direct/DIRect.c

```diff
@@ -19,7 +19,7 @@
 /* | DIRECT is a method to solve problems of the form:                     | */
 /* |              min f: Q --> R,                                          | */
 /* | where f is the function to be minimized and Q is an n-dimensional     | */
-/* | hyperrectangle given by the the following equation:                   | */
+/* | hyperrectangle given by the following equation:                   | */
```

Comment:
we can instead update to https://github.com/stevengj/nlopt/commit/f8692f86b5269112b3760704093d0600adb033e7, which includes this and other typos

Reply:
Fallback, please review again